notebooks/tutorial.ipynb

```
@@ -0,0 +1,584 @@
+   "outputs": [],
+   "source": [
+    "import nest_asyncio\n",
+    "nest_asyncio.apply()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30e66c894fb4cfd5",
+   "metadata": {},
+   "source": [
+    "To strike the balanace between speed, cost, anc quality, we recommend using OpenAI's `4o-mini` model; make sure your `.env` file contains this line:\n",
```

Pylint (3.3.8)

[error] 'await' should be used within an async function

(E1142)